compose/internal/composeplugin/composeplugin.go

```diff
@@ -99,16 +70,16 @@ func (wrapper *PluginWrapper) Pull(ctx context.Context, workingDir, host, projec
 			return err
 		}
 
-		log.Printf("[INFO] [compose,internal,composeplugin] [message: Stack pull successful]")
-		log.Printf("[DEBUG] [compose,internal,composeplugin] [output: %s]", output)
+		log.Printf("[INFO] [docker compose] [message: Stack pull successful]")
+		log.Printf("[DEBUG] [docker compose] [output: %s]", output)
 	}
 
 	return err
 }
 
 // Command exectue a docker-compose commanåd
```

Comment:
Not your fault, but while you're in here maybe fix this typo: "commanåd" to "command".

Reply:
oh man.  another one!  thanks :)   there were so many typos in this file.